README.md

```diff
@@ -1,26 +1,19 @@
 # NEWRELIC
 
-`We only support this integration for Artifactory versions above 7`
+`This integration is last tested with Artifactory 7.55.6 and Xray 3.69.3 versions.`
```

Comment:
Artifactory 7.55.6 is pretty old. Latest is 7.71.1. Similarly with Xray.

https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes

https://jfrog.com/help/r/jfrog-release-information/xray-release-notes

Reply:
Yes, I've tested it with Artifactory 7.71.4 and Xray 3.85.5 but forgot to update this. Nice catch. Thanks!